src/Dto/Feed.php

```diff
@@ -5,15 +5,16 @@
 namespace App\Dto;
 
 use App\Dto\Trait\BlameableTrait;
+use App\Dto\Trait\RelationsModifiedTrait;
 use App\Dto\Trait\TimestampableTrait;
 
 class Feed
 {
     use BlameableTrait;
     use TimestampableTrait;
+    use RelationsModifiedTrait;
 
     public ?array $configuration = [];
-    public ?string $slide = null;
-    public ?string $feedSource = null;
-    public ?string $feedUrl = null;
```

Comment:
Removing feedUrl is a breaking change. It is used here
https://github.com/os2display/display-client/blob/develop/src/data-sync/pull-strategy.js#L407

Reply:
Added